pdk/modules/ROOT/pages/policies-pdk-prerequisites.adoc

```diff
@@ -10,7 +10,7 @@ To begin using the Policy Development Kit (PDK), ensure you have the following p
 
 The Policy Development Kit (PDK) only supports Flex Gateway Policies. 
 
-The PDK tutorials assume that you have xref:flex-gateway.adoc[Downloaded Flex Gateway] and have some prior knowledge of the product.
+The PDK tutorials assume that you have xref:gateway::index.adoc[Downloaded Flex Gateway] and have some prior knowledge of the product.
```

Comment:
I'm thinking you should probably link to the download/installation page (flex-install.adoc).